docs/pages/project/contributing/contributing-model-relationships.md

```diff
@@ -222,70 +223,4 @@ Scope is defined and controlled via the `selectors` [Selectors](#structure-of-se
    - No relationship type (Kind) is inherently more important than the next one, so will not be any case
      of conflict
 
-## Relationship Types
```

Comment:
This is some of the most explanatory content in the page. Why is this being removed, @pandeyshubham03? @ritiksaxena124 @nwanduka?

Reply:
We already have the details for the Relationships kinds and subtypes in line 30. This part is a repetition.

Reply:
The repetition up top is the worse of the two.